homeassistant/components/husqvarna_automower/number.py

```diff
@@ -189,6 +185,7 @@ def __init__(
     ) -> None:
         """Set up AutomowerNumberEntity."""
         super().__init__(mower_id, coordinator)
+        self.coordinator = coordinator
```

Comment:
This is a coordinator entity which already has an attribute `coordinator`. We don't need to do this.